rossum/password.py

```diff
@@ -12,11 +12,10 @@ def cli() -> None:
 @click.password_option()
 @click.pass_context
 def change_command(ctx: click.Context, password: str) -> None:
-    if password:
```

Comment:
I suppose this statement is not needed because `password`/`password_option` options  require an argument